src/components/PopupWithConfirm.js

```diff
@@ -0,0 +1,30 @@
+import Popup from "./Popup";
+
+export default class PopupWithConfirm extends Popup {
+  constructor({ modalSelector }) {
+    super({ modalSelector });
+    this._modalForm = this._modalElement.querySelector(".modal__form");
+    this._submitButton = this._modalForm.querySelector(".modal__button-save");
+    this._submitButtonTextContent = this._submitButton.textContent;
+  }
+
+  setSubmitFunc(SubmitFunc) {
+    this._SubmitFunc = SubmitFunc;
+  }
+
+  setEventListeners() {
+    super.setEventListeners();
+    this._modalForm.addEventListener("submit", (evt) => {
+      evt.preventDefault();
+      this._SubmitFunc();
+    });
+  }
+
+  setLoading(isLoading, loadingText = "Saving...") {
```

Comment:
` "Saving..."`


In `PopupWithConfirm` it should be `Deleting...` here

Reply:
My remark hasn't been corrected https://github.com/bencorpus/se_project_aroundtheus/pull/1#discussion_r2029915022